README.md

```diff
@@ -76,6 +76,8 @@ fast.
 
 Learn more about [validating your AMP pages](https://www.ampproject.org/docs/guides/validate.html).
 
+Tutorial on [how to validate AMP pages](https://asknoypi.com/accelerated-mobile-pages-validator/).
```

Comment:
I added a tutorial link so first timers can understand how the validation works. If you are not really good in searching you will end up confused on how the validation takes place on Chrome. The tutorial contains images that may also help testers navigate the testing console on Chrome.